opencti-platform/opencti-front/src/private/components/common/stix_core_objects/StixCoreObjectSimulationResult.jsx

```diff
@@ -345,50 +452,110 @@ const StixCoreObjectSimulationResult = ({ id, type }) => {
       />
     );
   };
+
+  // Validation for Platforms
+  useEffect(() => {
+    if (platforms.length === 0) {
+      setPlatformError(t_i18n('This field should not be empty'));
```

Comment:
we don't do this way to set errors in form.. we should harmonize, maybe not use FormControl to be able to use our classical validators. What do you think @aHenryJard ?